services/google_drive_service.py

```diff
@@ -0,0 +1,38 @@
+from io import BytesIO
+import json
+import os
+from googleapiclient.discovery import build
+from googleapiclient.http import MediaIoBaseDownload
+from googleapiclient.errors import HttpError
+from google.oauth2.service_account import Credentials
+
+SERVICE_ACCOUNT_FILE = os.environ['GOOGLE_SERVICE_ACCOUNT_FILE']
+
+from logger import create_log
+
+logger = create_log(__name__)
+
+service_account_info=json.loads(SERVICE_ACCOUNT_FILE)
+scopes = ['https://www.googleapis.com/auth/drive']
+credentials = Credentials.from_service_account_info(service_account_info, scopes=scopes)
+
+drive_service = build('drive', 'v3', credentials=credentials)
```

Comment:
I left this as functions (instead of encapsulating it in a class) based on your comments -- this breaks from the other files in /services but I can see an argument for doing it either way. Let me know if this is what you were envisioning.

Reply:
Looks good!